.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -39,7 +39,7 @@ data class CIBuildModel (
                     functionalTests = listOf(
                             TestCoverage(TestType.quickFeedbackCrossVersion, OS.linux, JvmVersion.java7),
                             TestCoverage(TestType.quickFeedbackCrossVersion, OS.windows, JvmVersion.java7),
-                            TestCoverage(TestType.java9Smoke, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.quick, OS.linux, JvmVersion.java9),
```

Comment:
I think the `TestType` should be `platform`, given that the integration tests have been running with `forking`.

Reply:
Right. `forking` is needed here.

`java9Smoke` didn't run tests though, only integration tests.
Shall we introduce a new `TestType` for this? or could we go with `platform` and also run tests on Java 9?

Reply:
I wouldn't introduce a new `TestType`. Let's see if the unit tests pass on Java 9 (I fear some will fail...). We should just exclude whichever are failing and run unit tests on Java 9, too.